tests/postgres/derives.rs

```diff
@@ -58,6 +58,14 @@ enum ColorUpper {
     Blue,
 }
 
+#[derive(PartialEq, Debug, sqlx::Type)]
+#[sqlx(rename = "color_screaming_snake")]
+#[sqlx(rename_all = "screaming_snake_case")]
+enum ColorScreamingSnake {
```

Comment:
In https://github.com/launchbadge/sqlx/pull/304 tests are added for multiple databases, but it *seemed* like that's no longer necessary to do explicitly as of e5b6047009a425aec6fe939d7688db3723b58948 since that removed the other tests. Let me know if that's an incorrect assumption, and I can add tests for other databases.

Reply:
We don't have any standing policy right now dictating that everything must be tested on all databases. I think in general we want to keep the bar for landing contributions as low as possible while still having a minimal level of assurance that a given feature works. Not everyone is willing or able to write and run tests for all DB flavors.

For something like this that is pretty database-agnostic, I think is okay being tested on just one database. For PRs that may potentially have DB-specific issues like adding new types, we definitely want to cover all our (data)bases.

Reply:
That makes sense, thanks for the clarification!